internal/bootstrap/tenant_event_discovery.go

```
@@ -0,0 +1,150 @@
+// Copyright 2025 Lerian Studio. All rights reserved.
+// Use of this source code is governed by an Elastic License 2.0
+// that can be found in the LICENSE.md file.
+
+package bootstrap
+
+import (
+	"context"
+	"fmt"
+
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gci)